### PR TITLE
Refactor spaces formatting

### PR DESCRIPTION
**Pull request by zuoxingdong**
@christopherhesse I've extracted from #1319 for formatting-related fixes

### CODE REVIEW

File: gym/spaces/dict.py [hunk 1]
```diff
@@ -21,7 +21,7 @@ class Dict(Space):
             )),
             'rear_cam': spaces.Box(low=0, high=1, shape=(10, 10, 3)),
         }),
-        'ext_controller': spaces.MultiDiscrete((5, 2, 2)),
+        'ext_controller': spaces.MultiDiscrete([ [0,4], [0,1], [0,1] ]),
```
Comment (pzhokhov) [inline gym/spaces/dict.py]: I think this needs to be spaces.MultiDiscrete((5, 2, 2)) is the correct version, the other one would not work

**Comment by pzhokhov** [comment]
lgtm (other than minor issue with comment about the Dict space)

**Comment by zuoxingdong** [comment]
@pzhokhov fixed

**Comment by pzhokhov** [comment]
lgtm! @zuoxingdong please fix the merge conflicts (I realize that they are a consequence of my tardiness, so apologies for that - I'd fix them myself but that requires push access to your repo), and I'll merge.

**Pull request closed without merging**